### PR TITLE
Update the relationship docs to remove some references to a globals s…

### PR DESCRIPTION
**Pull request by bmac**
…tyle namespace

### CODE REVIEW

File: addon/-private/system/relationships/ext.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ export const RelationshipsClassMethodsMixin = Ember.Mixin.create({
     });
    ```
 
-    Calling `App.Post.typeForRelationship('comments')` will return `App.Comment`.
+    Calling `postRecord.constructor.typeForRelationship('comments')` will return `Comment`.
```
Comment (pangratz) [inline addon/-private/system/relationships/ext.js]: If I recall correctly, @fivetanley had some thoughts of not advertising `constructor` publicly? Just want to check...

Comment (pangratz) [inline addon/-private/system/relationships/ext.js]: Hmm, also it looks like `typeForRelationship` needs the `store` to be passed as 2nd argument...

File: addon/-private/system/relationships/ext.js [hunk 2]
```diff
@@ -209,8 +209,8 @@ export const RelationshipsClassMethodsMixin = Ember.Mixin.create({
     });
     ```
 
-    App.Post.inverseFor('comments') -> { type: App.Message, name: 'owner', kind: 'belongsTo' }
-    App.Message.inverseFor('owner') -> { type: App.Post, name: 'comments', kind: 'hasMany' }
+    postRecord.constructor.inverseFor('comments') -> { type: App.Message, name: 'owner', kind: 'belongsTo' }
+    messageRecord.constructor.inverseFor('owner') -> { type: App.Post, name: 'comments', kind: 'hasMany' }
```
Comment (pangratz) [inline addon/-private/system/relationships/ext.js]: `store` needs to be passed as 2nd argument...

**Comment by bmac** [comment]
Done @pangratz.

**Pull request merged**